docs/aqua-book/language/types.md

```diff
@@ -2,23 +2,52 @@
 
 ## Scalars
 
-Scalar types follow the Wasm IT notation.
+Scalar types in Aqua follow the Wasm IT notation.
 
 * Unsigned numbers: `u8`, `u16`, `u32`, `u64`
 * Signed numbers: `i8`, `i16`, `i32`, `i64`
 * Floats: `f32`, `f64`
 * Boolean: `bool`
 * String: `string`
-* Structures (product types): see [Structures](types.md#structures) below
-* Arrays: see [Collection Types](types.md#collection-types) below
 
-## Literals
+## Data Types
```

Comment:
Why is it separated from scalars above?

Reply:
Because generally speaking, they are not scalars. See https://stackoverflow.com/a/6628566
> Scalars are typically contrasted with compounds, such as arrays, maps, sets, structs, etc. A scalar is a "single" value - integer, boolean, perhaps a string - while a compound is made up of multiple scalars (and possibly references to other compounds).

And they are not scalars in Aqua compiler type system implementation: https://github.com/fluencelabs/aqua/blob/313502ecae4f0568af63158e6f518bc8e398953d/types/src/main/scala/aqua/types/Type.scala#L368